packages/expo-ota/ios/EXOta/EXEmbeddedManifestAndBundle.m

```diff
@@ -0,0 +1,47 @@
+//
+//  EXEmbeddedManifestAndBundle.m
+//  EXOta
+//
+//  Created by Michał Czernek on 25/10/2019.
+//
+
+#import "EXEmbeddedManifestAndBundle.h"
+
+@implementation EXEmbeddedManifestAndBundle {
+  NSDictionary *embeddedManifest;
+}
+
+-(NSDictionary *) readManifest
+{
+  if(embeddedManifest == nil)
+  {
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"shell-app-manifest" ofType:@"json"];
+    NSData *data = [NSData dataWithContentsOfFile:path];
+    embeddedManifest = [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error:nil];
```

Comment:
this causes the app to crash immediately if we don't have a `shell-app-manifest.json` embedded in the ios app. this isn't explained in readme and doesn't seem ideal - might make sense to just have app.json there and give a good error message if the user hasn't included it in their project